deepchem/feat/smiles_tokenizer.py

```diff
@@ -0,0 +1,369 @@
+# Requriments - transformers, tokenizers
+# Right now, the Smiles Tokenizer uses an exiesting vocab file from rxnfp that is fairly comprehensive and from the USPTO dataset.
+# The vocab may be expanded in the near future
+
+import collections
+import logging
+import os
+import re
+import numpy as np
+import pkg_resources
+import typing
+from typing import List
+from transformers import BertTokenizer
+from logging import getLogger
+
+logger = getLogger(__name__)
+
+
+try:
+  from transformers import BertTokenizer
+except ModuleNotFoundError:
+  logger.warning("HuggingFace transformers is not available. Please install using 'pip install transformers' to use the SmilesTokenizer")
+  print("HuggingFace transformers is not available. Please install using 'pip install transformers' to use the SmilesTokenizer.")
```

Comment:
Could you remove the print statement here? (For production use cases, we should make sure we only use `logger` to output statements). Also, no need for the `pass statement`

Reply:
will do

Reply:
done